pydantic_core/core_schema.py

```diff
@@ -2200,7 +2205,7 @@ class WithDefaultSchema(TypedDict, total=False):
     serialization: SerSchema
 
 
-Omitted = object()
+Omitted = Undefined
```

Comment:
We now have `Omitted` here and `Undefinded` defined in Rust.

Actually, I like the name `Omitted` more for the purpose we are using Undefined now.

I would like to rename `Undefinded` to `Omitted`.